lib/utils.js

```diff
@@ -274,6 +288,23 @@ exports.setCharset = function setCharset(type, charset) {
 };
 
 /**
+ * Helper function for creating a getter on an object.
+ *
+ * @param {Object} obj
+ * @param {String} name
+ * @param {Function} getter
+ * @private
+ */
+
+exports.defineGetter = function defineGetter(obj, name, getter) {
```

Comment:
I'm not sure if we really need a utility for this so +0 on this change

Reply:
Yea, I agree, no utility is necessary.

Reply:
Ok, I'll take it out then.
Although I just copied it from the request response instead of repeating.